src/app/routes.js

```diff
@@ -6,4 +6,18 @@ export function routes($routeProvider, $locationProvider) {
         requireBase: false,
         rewriteLinks: true,
     });
+
+    $routeProvider
+    .when("/", {
```

Comment:
use single quotes when writing javascript code, double quotes is meant to be used inside html context only